regenerate.sh

```diff
@@ -52,15 +65,30 @@ SOURCES=(
   ${WORKDIR}/grpc-proto/grpc/lb/v1/load_balancer.proto
   ${WORKDIR}/grpc-proto/grpc/lookup/v1/rls.proto
   ${WORKDIR}/grpc-proto/grpc/service_config/service_config.proto
+  ${WORKDIR}/grpc-proto/grpc/tls/provider/meshca/experimental/config.proto
   $(git ls-files --exclude-standard --cached --others "*.proto")
 )
-OPTS=Mgrpc/service_config/service_config.proto=/internal/proto/grpc_service_config
+# These options of the form 'Mfoo.proto=bar' instruct the codegen to use an
+# import path of 'bar' in the generated code when 'foo.proto' is imported in
+# one of the sources.
+MAPPINGS=(
+  "grpc/service_config/service_config.proto=/internal/proto/grpc_service_config"
+  "envoy/config/core/v3/config_source.proto=github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+)
+OPTS=""
+for i in "${MAPPINGS[@]}"
+do
+  OPTS+="M$i,"
+done
```

Comment:
I don't think this extra complexity is a good idea.  Just write it literally:

```
OPTS=Mgrpc/service_config/service_config.proto=/internal/proto/grpc_service_config,\
Menvoy/config/core/v3/config_source.proto=github.com/envoyproxy/go-control-plane/envoy/config/core/v3
```

(Also, the code above leaves a trailing comma which apparently works now, but doesn't seem like a good thing to be doing.)

Reply:
Done.